examples/flutter_gallery/lib/gallery/backdrop.dart

```diff
@@ -56,26 +60,32 @@ class _TappableWhileStatusIsState extends State<_TappableWhileStatusIs> {
 
   void _handleStatusChange(AnimationStatus status) {
     final bool value = widget.controller.status == widget.status;
-    if (_active != value) {
-      setState(() {
-        _active = value;
-      });
+    if (_value != value) {
+      setState(() { _value = value; });
     }
   }
 
   @override
   Widget build(BuildContext context) {
-    return new AbsorbPointer(
-      absorbing: !_active,
-      // Redundant. TODO(xster): remove after https://github.com/flutter/flutter/issues/17179.
-      child: new IgnorePointer(
-        ignoring: !_active,
-        child: widget.child
-      ),
-    );
+    return widget.builder(_value);
```

Comment:
I was already having a bit of trouble reading the indirection of the existing code. If the issue is the rendering cost, can we just wrap the whole thing in an `Offstage` based on the same flag?

Reply:
@xster : do you have more comments on this after we discussed this noon?